client/app/src/main/java/com/example/criengine/UserProfileActivity.java

```diff
@@ -0,0 +1,14 @@
+package com.example.criengine;
+
+import android.os.Bundle;
+public class UserProfileActivity extends ProfileActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        setContentView(R.layout.activity_user_profile);
+        super.onCreate(savedInstanceState);
+
+        // new UI components

```

Comment:
I'm assuming these are yet to be completed activities?

Also side not: Remember to PR titles to be in the form of "feat: <>" or "fix: <>" etc. Its more descriptive as to what is going on and will be easier to find if we need to later on

Reply:
yea i'm not sure if i'd be able to finish that by today (need to work on 379 a bit) but it's some placeholder for when I get to it next week

Reply:
Sounds good.
We did agree to only submit PR's when they were in their final working state so I'll assign the WIP tag for this.

Everything else is just a refactor so I suppose its nothing urgent.

Reply:
this PR is done, and is in a working state (none of that code is run)

Reply:
i supoooose soo...

I guess in the future we should avoid adding in files that don't relate to the purpose of the PR 🤷 (the empty activities)